src/tools/wat2wasm.cc

```diff
@@ -80,6 +85,12 @@ static void ParseOptions(int argc, char* argv[]) {
   parser.AddOption('o', "output", "FILE",
                    "Output wasm binary file. Use \"-\" to write to stdout.",
                    [](const char* argument) { s_outfile = argument; });
+  parser.AddOption('p', "inputfile", "PATHFILE",
+                   "Input binary file. Use \"-\" to get data.",
+                   [](const char* argument) { s_binfile = argument; s_read_binfile = true; });
+  parser.AddOption('c', "customname", "CUSTOM",
+                   "Input custom section name. Use \"-\" to write custom section.",
```

Comment:
How about calling this `--append-section`?      (Since it can only append the end).

What does `Use \"-\" to write custom section` mean here?